psl/psl/tests/validation/relations/set_null_is_not_valid_on_mixed_fields.prisma

```diff
@@ -0,0 +1,32 @@
+model SomeUser {
+  id      Int      @id
+  ref     Int
+  profile Profile?
+
+  @@unique([id, ref])
+}
+
+model Profile {
+  id       Int       @id
+  user     SomeUser? @relation(fields: [user_id, user_ref], references: [id, ref], onUpdate: SetNull, onDelete: SetNull)
+  user_id  Int?
+  user_ref Int
+
+  @@unique([user_id, user_ref])
+}
```

Comment:
This schema with one required referenced field in a  relation with the `SetNull` referential action was considered valid by Prisma. After this PR, it's not.

Reply:
👍🏾